ipatests/test_ipapython/test_certdb.py

```diff
@@ -284,3 +288,27 @@ def test_check_validity():
         nssdb.verify_server_cert_validity(CERTNICK, CERTSAN)
         with pytest.raises(ValueError):
             nssdb.verify_server_cert_validity(CERTNICK, 'invalid.example')
+
+
+@pytest.mark.parametrize("flags,trustflags, bijective ", [
+    (",,", TrustFlags(False, None, None, frozenset()), True),
+    ("u,u,u", TrustFlags(True, None, None, frozenset()), True),
+    ("p,p,p", TrustFlags(False, None, None, frozenset()), False),
+    (
+        "C,,",
+        TrustFlags(False, True, True, frozenset({x509.EKU_SERVER_AUTH})),
+        True
+    ),
+    ("CTu,Cu,Cu", TrustFlags(True, True, True, frozenset(
+        {
+            x509.EKU_SERVER_AUTH,
+            x509.EKU_CLIENT_AUTH,
+            x509.EKU_EMAIL_PROTECTION,
+            x509.EKU_CODE_SIGNING
+        }
+    )), True),
+])
```

Comment:
This test is a nice addition. Can you add one more for CT,C,C (e.g. has_key=False)? This would test external CA handling.